Foods.yaml

```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias"]
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "tacos"]
```

Comment:
Are tacos really that good though?

Reply:
Yes? :taco:

Reply:
They can have whatever within them! From simple, to complicated!